frontend/lib/norent/letter-builder/national-metadata.tsx

```diff
@@ -81,3 +82,11 @@ export const getNorentMetadataForUSState = (state: USStateChoice) => {
     legalAid: StateLegalAidProviders[state],
   };
 };
+
+/**
+ * Return a boolean determining whether a given zipcode is within
+ * our array of Los Angeles County zipcodes.
+ */
+export const zipCodeIsInLosAngeles = (zipCode: string) => {
+  return LosAngelesZipCodes.includes(zipCode);
+};
```

Comment:
I very slightly prefer `isZipCodeInLosAngeles` simply because it matches the naming convention of our other predicate functions (like your `isUserOutsideLA` in this same PR), it sounds more like a question when spoken aloud, and seeing "is" at the beginning of the name instantly makes my brain think "oh ok, this returns a boolean", but this is fine too.

Reply:
right right, good point.